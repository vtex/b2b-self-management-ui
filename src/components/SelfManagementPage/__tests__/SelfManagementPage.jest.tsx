import React from 'react'
import { SelfManagementPage } from '../'
import { render } from '@testing-library/react'

const mockOnClick = jest.fn()

jest.mock('@faststore/ui', () => ({
    Button: () => <button />
}))


it('Should render all information correctly', () => {
    const pageMockProps = {
        title: 'MockTitle',
        content: 'MockContent',
        buttonLabel: 'MockLabel',
        onClick: mockOnClick
    }

    const { container } = render(<SelfManagementPage {...pageMockProps} />)

    expect(container).toMatchSnapshot()
})