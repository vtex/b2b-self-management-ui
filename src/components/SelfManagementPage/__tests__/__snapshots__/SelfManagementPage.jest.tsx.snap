// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render all information correctly 1`] = `
<div>
  <section
    data-fs-self-management-section="true"
  >
    <h1
      data-fs-self-management-title="true"
    >
      MockTitle
    </h1>
    <code
      data-fs-self-management-code="true"
    >
      MockContent
    </code>
    <button />
  </section>
</div>
`;
